doc/source/whatsnew/v0.20.0.rst

```diff
@@ -1308,7 +1308,7 @@ Deprecations
 Deprecate ``.ix``
 ^^^^^^^^^^^^^^^^^
 
-The ``.ix`` indexer is deprecated, in favor of the more strict ``.iloc`` and ``.loc`` indexers. ``.ix`` offers a lot of magic on the inference of what the user wants to do. To wit, ``.ix`` can decide to index *positionally* OR via *labels*, depending on the data type of the index. This has caused quite a bit of user confusion over the years. The full indexing documentation is :ref:`here <indexing>`. (:issue:`14218`)
```

Comment:
Same as above

Reply:
so should I change back to the original "wit" in both instances and resubmit the pull request? Thanks.

Reply:
You can simply change it back and commit, which it looks like you've done